views/administration/classes_skipregistration.hbs

```diff
@@ -5,26 +5,24 @@
 
 {{#content "page"}}
 <p>
-    Du kannst auf dieser Seite im Namen deiner Schüler das Einverständnis zur <a
+    {{$t "administration.classes.label.youCanConsentToThisOnBehalfOfYourStudent" }}<a
         href="{{@root.theme.documents.specificFiles.privacyExemplary}}"
-        target="_blank">Datenschutzerklärung</a>
-    und zur <a href="{{@root.theme.documents.specificFiles.termsOfUseExemplary}}"
-        target="_blank">Nutzungsordnung</a> der {{@root.theme.short_title}} erklären. <br>
-    Durch den Klick auf <i>"Einverständnis im Namen der Schüler erklären"</i> bestätigst du, dass du das Einverständnis
-    der Schüler bzw. deren Eltern auf anderem Wege erhalten hast.
+        target="_blank">{{$t "administration.global.text.dataProtection" }}</a>
+    {{$t "administration.classes.text.andTo" }} <a href="{{@root.theme.documents.specificFiles.termsOfUseExemplary}}"
+        target="_blank">{{$t "administration.classes.label.termOfUse" }}</a> {{$t "administration.classes.text.ofThe" }} {{@root.theme.short_title}} {{$t "administration.classes.text.toExplain" }} <br>
+    {{$t "administration.classes.text.byClickingOn" }} <i>{{$t "administration.classes.button.declareConsentOnBehalfOfTheStudents" }}</i> {{$t "administration.classes.text.consentConfirmation" }}
```

Comment:
please but the whole text into one translation key (and add the theme.short_title as variable)

when you include the key you need to add {{{ brackets }}} , to not strip the html